code/datums/outfits/outfit.dm

```diff
@@ -21,6 +21,7 @@ var/global/list/outfits_decls_by_type_
 
 /decl/hierarchy/outfit
 	name = "Naked"
+	abstract_type = /decl/hierarchy/outfit
```

Comment:
This outfit is valid for selection

Reply:
It throws a very strange runtime comparing `Naked` against 0.